HISTORY.rst

```diff
@@ -67,4 +67,9 @@ History
 * Use v1 of the streaming api
 * Add custom vocabulary to async example
 * Add filter_profanity to async and streaming clients, examples, and documentation
-* Add remove_disfluencies to async client
\ No newline at end of file
+* Add remove_disfluencies to async client
+
+2.7.2
+------------------
+
+* Add language to async client
```

Comment:
We should clarify that this is the `language` job option. In the previous changelog notes it is more obvious that these are features of the client, but `language` is too generic a term for someone to infer that this is the job property and not just new wording of the async client.